src/main.py

```diff
@@ -1,16 +1,12 @@
 import sys
 import time
+import driver
 from datetime import datetime, timedelta
 from data.scoreboard_config import ScoreboardConfig
 from renderer.main import MainRenderer
-from rgbmatrix import RGBMatrix, RGBMatrixOptions
 from utils import args, led_matrix_options, stop_splash_service
 from data.data import Data
 import threading
-from sbio.dimmer import Dimmer
```

Comment:
These are now conditional imports if the driver thinks it's running on real hardware